packages/web-components/src/tab/tab.template.ts

```diff
@@ -0,0 +1,14 @@
+import { endSlotTemplate, FASTTab, startSlotTemplate, TabOptions } from '@microsoft/fast-foundation';
+import { ElementViewTemplate, html } from '@microsoft/fast-element';
+
+export function tabTemplate<T extends FASTTab>(options: TabOptions = {}): ElementViewTemplate<T> {
+  return html<T>`
+    <template slot="tab" role="tab" aria-disabled="${x => x.disabled}">
+      ${startSlotTemplate(options)}
+      <span class="tab-content"><slot></slot></span>
```

Comment:
Are we only forking the template here to accommodate the "semibold" hack? :(

Reply:
yes... i needed a third place to add content programmatically. ::before - for the activeIndicator gray state (stays on while indicator animates). ::after - for the animating indicator. Then I needed a third place to add the semi-bolded content (this tab-content class). If you can think of a better way let me know. I wasn't over the moon about this solution, but it was the best one I could see.